src/_constants/ActionTypes.js

```diff
@@ -53,3 +53,7 @@ export const UPDATE_VIDEO_LIST = 'UPDATE_VIDEO_LIST';
 
 export const WATCHLIST_FAVOR_ASSET = 'WATCHLIST_FAVOR_ASSET';
 export const WATCHLIST_UNFAVOR_ASSET = 'WATCHLIST_UNFAVOR_ASSET';
+
+export const ADD_TOKEN = 'ADD_TOKEN';
```

Comment:
Just one for 'UPDATE_TOKEN' ? Actions don't need to be too granular.
Now looking at favor and unfavor it could be one action too.

Don't take my word for it though, search for best practices and let's discuss.